src/tests/suite/dfs_par_test.c

```diff
@@ -739,75 +776,122 @@ file_atomicity_test_helper(test_arg_t *arg, int rf)
 		assert_non_null(oids_lo);
 	}
 
-	par_gather(PAR_COMM_WORLD, &oid1.hi, oids_hi, 1, PAR_UINT64, 0);
-	par_gather(PAR_COMM_WORLD, &oid1.lo, oids_lo, 1, PAR_UINT64, 0);
+	par_gather(PAR_COMM_WORLD, &foid1.hi, oids_hi, 1, PAR_UINT64, 0);
```

Comment:
Not your issue here but couldn't you do this in a single gather with 2 entries instead of 1?

Reply:
yea for sure. a lot of optimizations can be made like this in the parallel tests.
but haven't given this much time to cleanup.